synth.py

```diff
@@ -30,12 +30,12 @@
   name, 
   version,
   bazel_target=f"//google/appengine/{version}:appengine-{version}-nodejs")
-  s.copy(library, excludes=["package.json","linkinator.config.json", "system-test/fixtures/sample/src/index.js","system-test/fixtures/sample/src/index.ts"])
+  s.copy(library, excludes=["package.json","README.md"])
```

Comment:
Do you need to exclude `linkinator.config.json` until we ship the first release to npm?

Reply:
I kind of appreciate leaving it out, because then synthtool will automatically remove the extra links in linkinator in the next PR. And, if there's an error, then I know to fix it upstream!